pyproject.toml

```diff
@@ -20,7 +21,7 @@ keywords=["shadow", "finder", "locator", "map"]
 "Bug Tracker" = "https://github.com/bellingcat/ShadowFinder/issues"
 
 [tool.poetry.scripts]
-shadowfinder = "shadowfinder.main:main_entrypoint"
+shadowfinder = "shadowfinder.__main__:main"
```

Comment:
I renamed the function `main_entrypoint()` to `main()` and the file `main.py` to `__main__.py` to better align the project with what has pretty much become the Python standard (you can see this same pattern in [the official `__main__` top-level code env docs here](https://docs.python.org/3.12/library/__main__.html)). 

`__main__.py` has special meaning, and is almost always used to expose a command-line interface for a package (see [`__main__.py` in Python Packages](https://docs.python.org/3.12/library/__main__.html#main-py-in-python-packages)).